README.md

```diff
@@ -70,33 +70,39 @@ To achieve this MLOS provides:
 ## Python only installation
 
 Some of the examples require only the installation of the mlos Python library. These examples do not use the shared memory infrastructure.
-To use this simplified installation, it's recommended to use the Anaconda python distribution and create a new conda environment:
+
+
+First, download the mlos code using git:
 
 ```
-$ conda create -n mlos_environment
-$ conda activate mlos_environment
+$ git clone git@github.com:microsoft/MLOS.git
```

Comment:
I think we discussed this earlier, but I don't know if we came to a conclusion.

I'm thinking we should standardize on https git remote urls for now.

Reply:
that's ... in another PR? I'll fix it here.